templateDataGenerator.go

```diff
@@ -288,6 +290,17 @@ func generateTemplateData(params Params, currentReplicas int, gitSource, gitOwne
 		data.OverrideDefaultWhitelist = len(params.WhitelistedIPS) > 0
 		data.NginxIngressWhitelist = strings.Join(params.WhitelistedIPS, ",")
 
+	case "apigee":
+		data.ServiceType = "ClusterIP"
+		data.UseNginxIngress = true
+		data.UseGCEIngress = false
+		data.UseDNSAnnotationsOnIngress = true
+		data.UseDNSAnnotationsOnService = false
+		data.UseCloudflareProxy = false
+		data.UseBackendConfigAnnotationOnService = false
+		data.LimitTrustedIPRanges = false
```

Comment:
Doesn't apigee have a set of ip ranges we can check?

Reply:
Need to check, but we are use certificate validation so I don't think there is a need combine both.